utility/src/main/java/org/oppia/util/parser/SvgDrawableTranscoder.kt

```diff
@@ -0,0 +1,21 @@
+package org.oppia.util.parser
+
+import android.graphics.drawable.PictureDrawable
+import com.bumptech.glide.load.Options
+import com.bumptech.glide.load.engine.Resource
+import com.bumptech.glide.load.resource.SimpleResource
+import com.bumptech.glide.load.resource.transcode.ResourceTranscoder
+import com.caverock.androidsvg.SVG
+
+/** SvgDrawableTranscoder converts SVG to PictureDrawable. */
+class SvgDrawableTranscoder : ResourceTranscoder<SVG?, PictureDrawable?> {
+  override fun transcode(
+    toTranscode: Resource<SVG?>,
+    options: Options
+  ): Resource<PictureDrawable?>? {
```

Comment:
What if we encoded from SVG to Picture, instead? Or, do we have to encode to a ``Drawable``? I don't see explicit value in using ``Drawable`` here since we just copy it later, and we don't seem to use ``Drawable``s for bitmaps until the final step (rendering it).

Reply:
Glide is using PictureDrawable so we need to pass drawable.

Reply:
Update from PictureDrawable to Picture.